sdcm/utils/gce_utils.py

```diff
@@ -28,13 +28,21 @@
 
 GceDriver = get_driver(Provider.GCE)  # pylint: disable=invalid-name
 
+# The keys are the region name, the value is the available zones, which will be used for random.choice()
+SUPPORT_REGIONS = {
+    # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
+    # choose zones c and d twice as often as zone b
+    'us-east1': 'bccdd',
+    'us-west1': 'abc'}
+
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
-        # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
-        return f"{region}-{random.choice('bccdd')}"  # choose zones c and d twice as often as zone b
-    return region
+    for support_region, zone_string in SUPPORT_REGIONS.items():
+        if region == support_region:
```

Comment:
@amoskong , isn't it suppose to be
```suggestion
        if region.startswith(support_region)
```

Reply:
Also one more question ? do we need to validate region_name+datacenter cases ? if that is the case we need to redo entire logic here

Reply:
i think, in this case, the region and dc is same.

Reply:
@dkropachev no, we should have only region part here, the function is called `append_zone` we append availability zone suffix to the region name